### PR TITLE
Pdf

### PR DESCRIPTION
**Pull request by stevenwilkin**

### CODE REVIEW

File: features/step_definitions/record_of_guidance_steps.rb [hunk 1]
```diff
@@ -6,3 +6,9 @@
 
   expect(AppointmentSummary.count).to eql(1)
 end
+
+Then(/^a record of guidance document is created$/) do
+  expect(page.response_headers['Content-Type']).to eql('application/pdf')
+  text = PDF::Inspector::Text.analyze(page.source).strings.join
```
Comment (gjvis) [inline features/step_definitions/record_of_guidance_steps.rb]: Race condition here (test-runner vs application), we can sort it out in a future PR if it becomes an issue.

File: features/step_definitions/record_of_guidance_steps.rb [hunk 2]
```diff
@@ -6,3 +6,9 @@
...
+  expect(text).to include('Joe Bloggs')
```
Comment (gjvis) [inline features/step_definitions/record_of_guidance_steps.rb]: Race condition here (test-runner vs application), we can sort it out in a future PR if it becomes an issue.

**Comment by stevenwilkin** [comment]
Rebased

**Comment by gjvis** [comment]
:+1: :ship:

**Pull request merged**